src/Components/edit/Edit.tsx

```diff
@@ -0,0 +1,101 @@
+import axios from "axios";
+import { FormikState, useFormik } from "formik";
+import React, { useEffect } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+
+type Employee = {
+  id: string;
+  firstName: string;
+  lastName: string;
+  phone: string;
+};
+
+export function Edit(): React.JSX.Element {
+  const { state } = useLocation();
+  const navigate = useNavigate();
+
+  const id = state.id;
+  const URI = `https://localhost:7189/api/employees/${id}`;
+
+  let employee: Employee = {
+    id: "",
+    firstName: "",
+    lastName: "",
+    phone: "",
+  };
+
+  const formik = useFormik<Employee>({
+    initialValues: employee,
+
+    onSubmit: (values) => {
+      axios
+        .put(URI, values)
+        .then((response) => {
+          if (response.status === 204) {
+            alert("Successfully updated employee!");
```

Comment:
alert is not user friendly. Use https://primereact.org/toast/ instead

Reply:
Done.